### ISSUE TITLE
Cannot op user from a fallback session server

### ISSUE THREAD

**Issue by alex-huff** [Unmojang/Drasl#2]
When adding "https://sessionserver.mojang.com" to `FallbackSessionServers`, the Minecraft server can authenticate both custom Drasl users, and premium Microsoft accounts. The Minecraft server cannot, however, op premium accounts.

This is because when `/op (player name)` executed, the Minecraft server uses the `/profiles/minecraft` endpoint of the `ServicesServer` to determine the UUID of who to make an operator. Adding an option for `FallbackServiceServers` could enable this to work.

**Comment by evan-goode** [comment]
Thanks for reporting, surprised anyone is using this yet haha. Seems like adding `FallbackServiceServers` will be an easy fix; I'll implement this when I get a chance. The config will probably look something like

```
[[FallbackServer]]
Nickname = "mojang"
SessionURL = "https://sessionserver.mojang.com"
ServicesURL = "https://api.mojang.com"
```

I've been busy lately with a new job, but I'm hoping to flesh this project out more soon with some more features and proper documentation. Expect breaking API/config changes.

**Comment by alex-huff** [comment]
> Seems like adding `FallbackServiceServers` will be an easy fix; I'll implement this when I get a chance. The config will probably look something like
> 
> ```
> [[FallbackServer]]
> Nickname = "mojang"
> SessionURL = "https://sessionserver.mojang.com"
> ServicesURL = "https://api.mojang.com"
> ```
Yeah this would be great. I have a temporary workaround, so don't feel rushed. I can't reasonably get my friends to use this until your PR gets merged in either PolyMC or PrismLauncher anyway :(

**Comment by evan-goode** [comment]
Ok, I think this is fixed as of 6ba5367620c71dbf5169732389566d8e88de9762. I still need to implement support for fallback API servers for some of the other routes, but `/op` at least should work.
